ietf/group/views.py

```diff
@@ -437,6 +438,39 @@ def prepare_group_documents(request, group, clist):
 
     return docs, meta, docs_related, meta_related
 
+def get_leadership(group_type):
+    people = Person.objects.filter(
+        role__name__slug='chair',
+        role__group__type=group_type,
+        role__group__state__slug__in=('active', 'bof', 'proposed')).distinct()
+    leaders = []
+    for person in people:
+        parts = person.name_parts()
```

Comment:
Is this needed vs something like `person.plain_name()`? It'd be nice not to add to the variety of ways we handle names in various places. I guess this is to produce a "last name-first" list so maybe there's not much choice if that's needed for the Secretariat's purposes.

Reply:
Yes. The list is sorted by last name to match the way the meeting badges are sorted.